src/SablierV2LockupTranched.sol

```diff
@@ -190,18 +190,17 @@ contract SablierV2LockupTranched is
             return 0;
         }
 
-        // If the end time is not in the future, return the deposited amount.
```

Comment:
The "not" should not have been removed.

An `endTime` lower than `currentTime` is an end time that IS in the past, which means that it IS NOT in the future.